_rules/80f0bf.md

```diff
@@ -0,0 +1,112 @@
+---
+name: No auto-play audio
+test_type: composite
+
+description: |  
+  This rule checks that auto-play audio does not last for more than 3 seconds, or it has a control mechanism.
+
+success_criterion: 
+- 1.4.2 # Audio Control
+
+atomic_rules:
+- 95e043
+- aaa1bf
+
+authors:
+- Anne Thyme Nørregaard
+- Bryn Anderson
+---
+
+## Applicability
+
+This rule applies to any HTML `<audio>` `<video>` or `<source>` elements, with a `src` attribute referencing content with a duration of more than 3 seconds or with a `loop` attribute equal to `true`, that has an `autoplay` attribute equal to `true`, and that has either a `paused` or `muted` attribute equal to `false`.
+
+## Expectation
+
+For each test target, the outcome of at least one of the following rules is passed:
+- Auto-play audio does not last more than 3 seconds
+- Auto-play audio has a control mechanism
+ 
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility Support
+
+There are no major accessibility support issues known for this rule.
+
```

Comment:
[SC 1.4.2 is part of the non-interference requirement](https://www.w3.org/TR/WCAG21/#cc5), don't you think that's an information of importance and that it should me mentionned at least in the composite rule ?

Reply:
Non-interference is a SC in its own right. Not sure how to handle it here or if it should be handled in this rule. I suppose that we could have a composite rule for Non-interference that includes this composite rule... Surely someone has brought this up at some point.